CONTRIBUTING.md

```diff
@@ -37,17 +44,51 @@ Do not hesitate to create an issue or pull request directly on the
 ## Implement Features
 
 The Taipy team manages its backlog in private. Each issue that will be done during our current sprint is
-attached to the `current sprint`. Please, do not work on it, the Taipy team is on it.
+attached to the "current sprint". Please, do not work on it, the Taipy team is on it.
```

Comment:
We currently do not have the `current sprint` label. Should we keep this or add the label?

Reply:
I'm not sure if we can automate the labels based on the status (the column name)

Reply:
Otherwise, we can just say that the issues with an assignee are already engaged so it is not available anymore for a contribution. That's simpler and it does include both internal and external contributions.

No?

Reply:
That would be better, yes